### DISCUSSION TITLE
Thank you for the best graphics library

### DISCUSSION THREAD

**Discussion by mike-rankin** [Bodmer/TFT_eSPI#1371] [Q&A - General]
Not really a discussion but just a quick message to say thank you for the best graphics library I've ever used. I started out with the Adafruit library but found the refresh rate way too slow. TFT_eSPI is crazy fast and easy for me who is not a fan of programming. My project is a bedside clock using Adafruit libraries for all the sensors except for the graphics library.

https://github.com/mike-rankin/ESP32_Desktop_Clock

**Comment by Bodmer** [comment]
Thanks for the encouraging comments,, and thanks for sharing your project.
